src/components/entries/Select.js

```diff
@@ -70,7 +70,7 @@ function Select(props) {
   return (
     <div class="bio-properties-panel-select">
       <label for={ prefixId(id) } class="bio-properties-panel-label">
-        <Tooltip value={ tooltip } labelId={ prefixId(id) }>
+        <Tooltip value={ tooltip } forId={ id } element={ props.element }>
```

Comment:
Changed `labelId` to `forId` to serve a more general purpose, trigger element doesn't need to be a label